ibm/data_source_ibm_is_security_groups.go

```diff
@@ -0,0 +1,838 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"reflect"
+	"time"
+
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	// "github.ibm.com/ibmcloud/vpc-go-sdk/vpcv1"
+)
+
+func dataSourceIBMIsSecurityGroups() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceIBMIsSecurityGroupsRead,
+
+		Schema: map[string]*schema.Schema{
+			"resource_group": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "resource group identifier.",
+			},
+			"vpc_id": &schema.Schema{
```

Comment:
vpc

Reply:
Actually `vpc` exists, thus would create redundancy with the existing variable set.